core/bin/system-constants-generator/src/main.rs

```diff
@@ -6,14 +6,13 @@ use multivm::{
     vm_latest::constants::MAX_PUBDATA_PER_BLOCK,
 };
 use serde::{Deserialize, Serialize};
-use zksync_types::{
-    zkevm_test_harness::zk_evm::zkevm_opcode_defs::{
-        circuit_prices::{
-            ECRECOVER_CIRCUIT_COST_IN_ERGS, KECCAK256_CIRCUIT_COST_IN_ERGS,
-            SHA256_CIRCUIT_COST_IN_ERGS,
-        },
-        system_params::MAX_TX_ERGS_LIMIT,
+use zkevm_test_harness_1_3_3::zk_evm::zkevm_opcode_defs::{
```

Comment:
uh interesting.. this is computing prices from the 'old' proof system.
we should upgrade it, to use the stuff from geometry_config and new proof system.

Also - you can probably depend directly on the zkevm_opcode_defs 1.3.2